test/utils/bytecode.hpp

```diff
@@ -550,16 +550,17 @@ inline std::string decode(bytes_view bytecode)
         {
             s += std::string{" + OP_"} + name;
 
-            if (opcode >= OP_PUSH1 && opcode <= OP_PUSH32)
+            if (size_t imm_data_size = evmone::instr::traits[opcode].immediate_size;
```

Comment:
```suggestion
            if (const size_t imm_data_size = evmone::instr::traits[opcode].immediate_size;
```

Reply:
I also think we could name it just `imm_size` unless it is already commonly named `imm_data` in other places.

Reply:
OK for naming. `imm_size` cannot be `const`. It's changed below.